usaspending_api/transactions/loader_functions.py

```diff
@@ -0,0 +1,52 @@
+from re import search as regex
+from pathlib import Path
+from typing import List, Tuple
+
+from usaspending_api.common.retrieve_file_from_uri import RetrieveFileFromUri
+
+
+def store_ids_in_file(id_iter: List, file_name: str = "temp_file") -> Tuple[Path, int]:
+    """Store the provided values in a file
+        returning filepath and count of ids
+    """
+    total_ids = 0
+    file_path = Path(file_name)
+
+    with open(str(file_path), "w") as f:
+        for id_string in id_iter:
+            if not id_string:
```

Comment:
Couldn't this be simplified to `if id_string is not None` and then the contents of the "for"? Avoiding the use of `continue` when it isn't needed.

Reply:
I try to think in terms of guard clauses https://refactoring.guru/replace-nested-conditional-with-guard-clauses and place the least-amount of code nested under the conditional